acquia_lift_profiles/js/acquia_lift_profiles.debug.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Defines JavaScript functions needed by the Acquia Lift debugger in
+ * order to provide customer profile information.
+ */
+
+(function ($, Drupal) {
+
+      function getFromCookie ( query )
```

Comment:
Drupal already includes jQuery.cookie 1.0 plugin.  You can include it in the requirements for the library and then utilize $.cookie(query) wherever you were using getFromCookie(query).